README.rst

```diff
@@ -38,6 +38,7 @@ and then kill the processes::
 
 Installation of QueueServer from source::
 
+  sudo apt install redis
```

Comment:
It's for debian-like system. Should we add more instructions for `yum`/`dnf` installation steps? (I assume no `zypper`, but we may need to also cover Arch Linux).

Reply:
As an Arch Linux user I think having the package name is enough and you can translate for your distro. I guess it could be more generic - use your package manager to install redis?

Reply:
I think so, but maybe leaving the example for Ubuntu as well.

Reply:
Looking into the future, we are designing a package potentially useful by non-NSLS-II Bluesky users. So having installation instructions for as many systems as possible may be useful. At least this could be a reason to keep already existing instructions, just group them properly.